tests/lora/test_lora_layers_old_backend.py

```diff
@@ -311,71 +355,12 @@ def test_stable_diffusion_xformers_attn_processors(self):
         image = sd_pipe(**inputs).images
         assert image.shape == (1, 64, 64, 3)
 
-    @unittest.skipIf(not torch.cuda.is_available(), reason="xformers requires cuda")
```

Comment:
Don't think we need to keep this test because it uses attention processors. Objections?